src/buffer/out/textBuffer.cpp

```diff
@@ -1430,12 +1430,13 @@ const til::point TextBuffer::GetGlyphStart(const til::point pos, std::optional<t
 }
 
 // Method Description:
-// - Update pos to be the end of the current glyph/character. This is used for accessibility
+// - Update pos to be the end of the current glyph/character.
 // Arguments:
 // - pos - a COORD on the word you are currently on
+// - accessibilityMode - this is being used for accessibility; make the end exclusive.
 // Return Value:
 // - pos - The COORD for the last cell of the current glyph (exclusive)
-const til::point TextBuffer::GetGlyphEnd(const til::point pos, std::optional<til::point> limitOptional) const
+const til::point TextBuffer::GetGlyphEnd(const til::point pos, bool accessibilityMode, std::optional<til::point> limitOptional) const

```

Comment:
The amount of arguments our `TextBuffer` member functions have are a bit of a horror story ngl.
Why does "accessibility mode" mean "become exclusive" here? Don't you mean "bool exclusivePosition"?

Reply:
Yeeeeaaaaah. So, a11y and selection have a lot of similarities and very small/important differences. Some highlights include...
| | Selection | A11y |
|--|--|--|
| endpoint representation | both endpoints are inclusive; start <= end | inclusive start; exclusive end; start <= end |
| definition of a word | a run of legible characters | run of legible characters + run of whitespace after it (until next word starts) |

I just chose to name the parameter `accessibilityMode` to make it clear what and when we're only doing some extra work for a11y.